<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>overlay_editor</class>
 <widget class="QDialog" name="overlay_editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Overlay</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDoubleSpinBox:disabled {background-color: LightGray;}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QComboBox" name="material"/>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="edit_style">
     <property name="text">
      <string>Edit Style</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QTabWidget" name="tab_widget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="powder_tab">
      <attribute name="title">
       <string>Powder</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>No Custom Settings for Powder</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="laue_tab">
      <attribute name="title">
       <string>Laue</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="laue_min_energy_label">
         <property name="text">
          <string>Min Energy:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="laue_min_energy">
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <double>10000.000000000000000</double>
         </property>
         <property name="value">
          <double>5.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="laue_max_energy_label">
         <property name="text">
          <string>Max Energy:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="laue_max_energy">
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <double>10000.000000000000000</double>
         </property>
         <property name="value">
          <double>35.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mono_rotation_series_tab">
      <attribute name="title">
       <string>Mono Rotation Series</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="type">
     <item>
      <property name="text">
       <string>Powder</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Laue</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mono Rotation Series</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="type_label">
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="material_label">
     <property name="text">
      <string>Material:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>material</tabstop>
  <tabstop>type</tabstop>
  <tabstop>edit_style</tabstop>
  <tabstop>tab_widget</tabstop>
  <tabstop>laue_min_energy</tabstop>
  <tabstop>laue_max_energy</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>overlay_editor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>overlay_editor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
